bin/configurer/add-roles.sh

```diff
@@ -4,13 +4,22 @@ set -eu
 
 dir=$(dirname ${0})
 
-if [[ "${IDAM_VERSION:-tactical}" == 'tactical' ]]; then
-  ${dir}/utils/idam-add-role.sh caseworker-publiclaw-gatekeeper "Case Worker Public Law Gatekeeper"
-fi
+${dir}/utils/idam-add-role.sh "ccd-import"
+${dir}/utils/idam-add-role.sh "caseworker"
+${dir}/utils/idam-add-role.sh "caseworker-publiclaw"
 
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-solicitor
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-courtadmin
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-cafcass
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-gatekeeper
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-systemupdate
-${dir}/utils/ccd-add-role.sh caseworker-publiclaw-judiciary
+# User used during the CCD import and ccd-role creation
+${dir}/utils/idam-create-caseworker.sh ccd.docker.default@hmcts.net ccd-import CCD_Docker
```

Comment:
Any reason why args are not in quotes? Looks bit inconsistent.

Reply:
A `CCD_Docker` is meaningless here. Can we remove? I tend to hide things that does not matter.

Reply:
Done